movies_import/signals/handlers.py

```diff
@@ -0,0 +1,9 @@
+from django.db.models.signals import post_save
+from django.dispatch import receiver
+from movies_import.utils import import_movies_from_zip
+from movies_import.models import *
+
+
```

Comment:
Usually, signals are included in the `signals.py` file placed in the application root directory. It makes sense to split the file in multiple files inside a `signals` directory when the file gets bigger and harder to keep track of signals / navigate, but this is not the case.